Engines/Wine/Verbs/gdiplus_winxp/script.js

```diff
@@ -0,0 +1,65 @@
+const Wine = include("engines.wine.engine.object");
+const Resource = include("utils.functions.net.resource");
+const { CabExtract } = include("utils.functions.filesystem.extract");
+const { remove, cat, writeToFile } = include("utils.functions.filesystem.files");
+
+const Optional = Java.type("java.util.Optional");
+
+include("engines.wine.plugins.override_dll");
+
+/**
+ * Verb to install gdiplus (windows xp)
+ *
+ * @returns {Wine} Wine object
+ */
+class GDIPlusWinXP {
+    constructor(wine) {
+        this.wine = wine;
+    }
+
+    go() {
+        const wizard = this.wine.wizard();
+        const prefixDirectory = this.wine.prefixDirectory();
+        const system32directory = this.wine.system32directory();
+
+        const setupFile = new Resource()
+            .wizard(this.wizard())
```

Comment:
`this.wizard()` does not exist

Reply:
I got the same thing in adobe air verb

Reply:
What do you mean by you got the same thing?

Reply:
There is also this.wizard in adobe air verb ^^

Reply:
Then I most likely overlooked it. It shouldn't work because the verb classes don't define a `wizard` member field/method. `wine` does.

Reply:
I can update it in this PR too.

Reply:
Please do so

Reply:
While you are at it can you search the repository for `this.wizard()`? This shouldn't appear anywhere in the installers or the verbs.